README.md

```diff
@@ -82,7 +82,7 @@ How you configure DNS depends on your platform:
 
 ### More information and documentation
 
-For more deployment options, see the [deployment documentation][10].
+For more deployment options, including uninstalling Contour, see the [deployment documentation][10].
 
```

Comment:
I should have caught this before -- we should probably move the uninstall instructions into the readme. Makes more sense -- it's not a deployment option ;). I can do it when I rework Ralph's content if you like.

Reply:
We probably want it both places?  In any case, I'll let you fix it as you rework Ralph's stuff.